config/upload.js

```diff
@@ -0,0 +1,30 @@
+const multer = require('multer');
+const path = require('path');
+
+const storage = multer.diskStorage({
+  destination: (req, file, callBack) => {
+    callBack(null, '../public/uploads/images/');
```

Comment:
@kimberlym4488 This path needs to be combined with `__dirname`. I'd also like  to keep `uploads` separate from our `public` assets. So can you move the `uploads` folder up a directory so it's at the root of the project? Then you'll set it up as a second `static` folder in the server js. (copy the line for the `public` folder in the server js and rename `public` to `uploads`).

Updated like should look like this after moving the uploads folder.
```
callBack(null, path.join(__dirname, '../uploads/images/'));
```

Reply:
Yay it worked! Now I can  get the response routed back to display on idea detail. Thanks so much!

Reply:
- [ ] Uploads displaying properly.
- [ ] MD's showing file location but not displaying. Tried this solution:
 `   content: `<object data="${req.file.filename}" type="text/html">${req.file.filename}</object>`
- [ ] Links not uploading (no req.file found) Tried to change  ` upload.single('link') ` to 'none' with no success. 
![image](https://user-images.githubusercontent.com/92805933/158109561-15814e6c-80db-4663-9b67-5fe62b118374.png)

Any help is greatly appreciated!

Reply:
Let's tackle this in a future PR @kimberlym4488 can you make an issue for it?